src/orca/services/base/client_factory.py

```diff
@@ -2,39 +2,37 @@
 
 from abc import ABC, abstractmethod
 from functools import cached_property
-from typing import Any, Generic, Type, TypeVar
+from typing import Any, ClassVar, Generic, Type, TypeVar
 
 from pydantic.dataclasses import dataclass
 from typing_extensions import Self
 
-from orca.services.base.config import BaseServiceConfig
+from orca.services.base.config import BaseConfig
 
 ClientClass = TypeVar("ClientClass", bound=Any)
 
-ServiceConfig = TypeVar("ServiceConfig", bound=BaseServiceConfig)
+ConfigClass = TypeVar("ConfigClass", bound=BaseConfig)
 
 
 @dataclass(kw_only=False)
-class BaseClientFactory(ABC, Generic[ClientClass, ServiceConfig]):
-    """Base factory for constructing clients."""
+class BaseClientFactory(ABC, Generic[ClientClass, ConfigClass]):
+    """Base factory for constructing clients.
+
+    Class Variables:
+        connection_env_var: The name of the environment variable whose
```

Comment:
Are the Class Variables `config_class` and `client_class`?

Reply:
This was addressed in #11